### PR TITLE
Added missing translation keys in DE language

### PR DESCRIPTION
**Pull request by basilebong**
My PR is a:

💅 Enhancement

Main update on the: 
**Admin**

I added the missing german translation keys, resolving #1620

### CODE REVIEW

**Comment by lauriejim** [comment]
Thank you @basilebong for this PR !
I will review all translation PRs at the end of next week.

File: packages/strapi-admin/admin/src/translations/de.json [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
     "app.components.HomePage.welcome": "Willkommen an Bord!",
+    "app.components.HomePage.welcome.again": "Willkommen",
     "app.components.HomePage.create": "Erstelle deinen ersten Content-Typ",
     "app.components.HomePage.welcomeBlock.content": "Wir freuen uns, dich als Mitglied der Community zu haben. Wir sind offen für Feedback, senden uns einfach eine direkt Nachricht an\u0020",
     "app.components.HomePage.welcomeBlock.content.issues": "Fehler",
     "app.components.HomePage.welcomeBlock.content.raise": "\u0020oder erhöhen\u0020",
     "app.components.HomePage.createBlock.content.first": "Das\u0020",
     "app.components.HomePage.createBlock.content.second": "\u0020Plugin wird dir helfen, die Datenstruktur deiner Modelle zu definieren. Wenn du neu hier bist, empfehlen wir dir unsere\u0020",
     "app.components.HomePage.createBlock.content.tutorial": "\u0020Anleitung.",
+    "app.components.HomePage.welcomeBlock.content.again": "Wir hoffen, Sie machen Fortschritte bei Ihrem Projekt.... Lesen Sie das Neueste über Strapi. Wir geben unser Bestes, um das Produkt auf der Grundlage Ihres Feedbacks zu verbessern.",
```
Comment (Jan10) [inline packages/strapi-admin/admin/src/translations/de.json]: The current translation is based on "Du". Please use "Du" instead of "Sie".

**Comment by Jan10** [comment]
@basilebong thank you!
The current translation is based on "Du". Can you please use "Du" instead of "Sie"?

**Comment by basilebong** [comment]
Yes of course. I will do this later this evening. 😊 @Jan10

**Pull request merged**